examples/demo.rs

```diff
@@ -156,6 +156,14 @@ fn main() {
 
     let mut perf = PerfGraph::new();
 
+    {
+        #[cfg(not(target_arch = "wasm32"))]
+        let window = windowed_context.window();
+        let dpi_factor = window.scale_factor();
+        canvas.set_size(0, 0, dpi_factor as f32);
+        canvas.reset();
+    }
```

Comment:
Note: I had to copy this under the `Resized` event for scale to work correctly on resize